### PR TITLE
Add prettier

### PR DESCRIPTION
**Pull request by etrobert**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "build": "snowpack build",
     "lint": "eslint src/",
     "deploy": "echo 'taskgraph.net' > build/CNAME && gh-pages -d build/",
-    "test": "jest"
+    "test": "jest",
+    "format": "prettier -w ."
```
Comment (justinline) [inline package.json:13]: question: this will format all files? is that intended?

**Comment by justinline** [comment]
Finished 🏁

File: package.json (hunk 1 above)
Comment (etrobert) [reply package.json:13]: All files except for those in `.prettierignore`

**Comment by justinline** [comment]
👍 Finished

**Pull request merged**